docs/mvc/baseController.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Base Controllers
+name: Overview
+parent: Model View Controller
+nav_order: 1
+---
+
+# Use base controllers
+
+Keep commonly used methods such as
+
+- getResourceBundle
+- getRouter
+- getModel
+- setModel
+- onNavBack
+- _or whatever method or property is being re-used in other controllers in the application_
+
+inside a base controller:
+
+    |-+ webapp
+      |-+ controller
+        |-- BaseController.js
+
+**webapp/controller/BaseController.js:**
+
+    sap.ui.define([
```

Comment:
sorry, missed this in the last review: please add a codefence à la ```js to get syntax highlighting:
```js
const hello = () => { world: "yay" }
```

Reply:
Done for all code snippets. Please check again!